examples/models/llama2/export_llama_lib.py

```diff
@@ -710,7 +710,7 @@ def _export_llama(modelname, args) -> str:  # noqa: C901
         # pyre-ignore: Undefined attribute [16]: Module `executorch.backends` has no attribute `qualcomm`
         backend_options = generate_htp_compiler_spec(use_fp16=False)
         # pyre-ignore: Undefined attribute [16]: Module `executorch.backends` has no attribute `qualcomm`
-        partitioner = QnnPartitioner(
+        partitioner = QnnPartitioner(  # noqa
```

Comment:
Why isn't this used? Feels like this `# noqa` is hiding a bug -- if this isn't being used, we should remove the creation of this QnnPartitioner. Or figure out where it was supposed to be used. Or, if it's meant to be ignored like this, it should have a comment explaining why, to go along with the `# noqa`.

Reply:
oh sorry it's a rebase issue, let me fix it